tests/foreman/cli/test_auth.py

```diff
@@ -296,10 +296,10 @@ def test_positive_session_preceeds_saved_credentials(
         assert LOGEDIN_MSG.format(admin_user['login']) in result[0]['message']
         # list organizations without supplying credentials
         sleep(70)
-        with pytest.raises(CLIReturnCodeError):
-            Org.with_user().list()
         result = Auth.with_user().status()
         assert LOGEDOFF_MSG.format(admin_user['login']) in result[0]['message']
+        with pytest.raises(CLIReturnCodeError):
```

Comment:
The BZ has been triaged, but doesn't look like it has been targeted for any release yet.

Since adding the BZ to the metadata will cause the testcase to skip execution if the bug is open, I think it would be a better approach here to execute this `with` block based on the BZ status.  There are examples in the framework of doing this - you just build an `if` statement that checks the BZ blocking status.

Then, until this BZ is fixed, the test can still run, and just won't check that a CLIReturnCodeError is raised when you try to list orgs without a valid token.